### PR TITLE
adding book suggestion

### PR DESCRIPTION
**Pull request by gonziesc**
## Summary
- Adding model book sugestion

## Trello Card
https://trello.com/c/GMjam3ks/29-crear-modelo-de-booksuggestion

### CODE REVIEW

File: db/migrate/20170913131914_create_active_admin_comments.rb [hunk 1]
```diff
@@ -1,17 +0,0 @@
-class CreateActiveAdminComments < ActiveRecord::Migration::Current
```
Comment (hdf1986) [inline db/migrate/20170913131914_create_active_admin_comments.rb]: Why is this delete here? maybe it's from another PR?

File: db/migrate/20170913170638_create_book_suggestion.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class CreateBookSuggestion < ActiveRecord::Migration[5.1]
+  def change
+    create_table :book_suggestions do |t|
+      t.string :editorial
```
Comment (hdf1986) [inline db/migrate/20170913170638_create_book_suggestion.rb]: can we set a default empty string to the non-required fields? also null/not-null for the rest of the columns

File: db/schema.rb [hunk 3]
```diff
@@ -46,6 +46,34 @@
     t.index ["reset_password_token"], name: "index_admin_users_on_reset_password_token", unique: true
   end
 
+  create_table "book_sugestions", force: :cascade do |t|
+    t.string "editorial"
```
Comment (hdf1986) [inline db/schema.rb]: Hmm your table is 2 times here, maybe the `g` issue? if that's the case remove the schema.rb file and regenerate it

File: db/migrate/20170913131914_create_active_admin_comments.rb (hunk 1 above)
Comment (gonziesc) [reply db/migrate/20170913131914_create_active_admin_comments.rb]: oh yes! it's in the other PR, i should merge

File: db/schema.rb (hunk 3 above)
Comment (gonziesc) [reply db/schema.rb]: got it

File: app/models/book_suggestion.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+class BookSuggestion < ApplicationRecord
+  belongs_to :user
```
Comment (hdf1986) [inline app/models/book_suggestion.rb]: Are we requiring this relation? check http://blog.bigbinary.com/2016/02/15/rails-5-makes-belong-to-association-required-by-default.html

File: db/migrate/20170913170638_create_book_suggestion.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+class CreateBookSuggestion < ActiveRecord::Migration[5.1]
+  def change
+    create_table :book_suggestions do |t|
+      t.string :editorial, :default => ''
```
Comment (hdf1986) [inline db/migrate/20170913170638_create_book_suggestion.rb]: Use the new sintax for the hashes

File: db/migrate/20170913170638_create_book_suggestion.rb [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+      t.float :price, :default => 0
```
Comment (hdf1986) [inline db/migrate/20170913170638_create_book_suggestion.rb]: Same here

File: db/migrate/20170913170638_create_book_suggestion.rb [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+      t.string :author
+      t.string :title
+      t.string :link
+      t.string :publisher, :default => ''
```
Comment (hdf1986) [inline db/migrate/20170913170638_create_book_suggestion.rb]: And here

**Pull request merged**